test/scrivener/paginator/ecto/query_test.exs

```diff
@@ -94,7 +94,6 @@ defmodule Scrivener.Paginator.Ecto.QueryTest do
       page =
         Post
         |> Post.published
-        |> order_by([p], desc: p.inserted_at)
```

Comment:
@enilsen16 while this change does fix the tests, we now have 2 tests with identical input:

https://github.com/enilsen16/scrivener_ecto/blob/04166f2e3370f7c5395090fa1209fd6081e07262/test/scrivener/paginator/ecto/query_test.exs#L79:L97

Turns out this particular test was already removed upstream [1], so I'm gonna remove after merging.

Btw, you should have write access to this repo (ping me if you don't) and so it's preferred if you create branches directly here and not from your fork - this way I could make the change directly on your branch :)

[1] https://github.com/drewolson/scrivener_ecto/commit/d076f5d7c3414eb3a6725b9a6660632a0478a0c8